### PR TITLE
Preparing relicensing to MIT-0

### PR DESCRIPTION
**Pull request by viktorklang**
Not to merge yet, just making sure I got all covered

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,10 +1,10 @@
 # Contributing to the Reactive Streams Project
 
-The Reactive Streams project welcomes contributions from anybody who wants to participate in moving this initiative forward. All code or documentation that is contributed will have to be covered by a waiver of all copyrights and other rights as detailed by the LICENSE and COPYING files at each repository root, the rationale for this is that the APIs defined by this project shall be freely implementable and usable by everyone.
+The Reactive Streams project welcomes contributions from anybody who wants to participate in moving this initiative forward. All code or documentation that is contributed will have to be covered the MIT No Attribution (SPDX: MIT-0) license, the rationale for this is that the APIs defined by this project shall be freely implementable and usable by everyone.
```
Comment (rkuhn) [inline CONTRIBUTING.md:3]: “covered _by_ the”

**Review by rkuhn — COMMENTED** [review]
Just found one typo, otherwise looks good to me.

File: CONTRIBUTING.md (hunk 1 above)
Comment (viktorklang) [reply CONTRIBUTING.md:3]: Fixed!

**Comment by viktorklang** [comment]
Note to self: enable DCO after merging: https://probot.github.io/apps/dco/

**Review by tbowling2020 — APPROVED** [review]

**Review by tbowling2020 — APPROVED** [review]

**Pull request merged**